### PR TITLE
[IMP] prestashop connector: translations

### PR DESCRIPTION
**Pull request by AurelijaNorvaisaite**
Due to changes in prestashop, from now on required format for fields with translations is:
<name><language id=2>Test</language></name>

Value and attr are no longer supported.

Additional changes to connector_prestashop_environment are added due to pre-commit.

### CODE REVIEW

File: connector_prestashop_environment/README.rst [hunk 1]
```diff
@@ -57,7 +57,6 @@ Contributors
 ~~~~~~~~~~~~
 
 * Florian da Costa <florian.dacosta@akretion.com>
-* Ugne Sinkeviciene <ugne@versada.eu>
```
Comment (psugne) [inline connector_prestashop_environment/README.rst]: Leave me! :D

Comment (AurelijaNorvaisaite) [reply connector_prestashop_environment/README.rst]: I also added you in contributor file

File: connector_prestashop_environment/static/description/index.html [hunk 2]
```diff
@@ -8,10 +8,11 @@
 
 /*
 :Author: David Goodger (goodger@python.org)
-:Id: $Id: html4css1.css 8954 2022-01-20 10:10:25Z milde $
+:Id: $Id: html4css1.css 9511 2024-01-13 09:50:07Z milde $
```
Comment (psugne) [inline connector_prestashop_environment/static/description/index.html:11]: @AurelijaNorvaisaite what's the change here? Was it generated automatically?

Comment (AurelijaNorvaisaite) [reply connector_prestashop_environment/static/description/index.html:11]: yes, it was changed via pre-commit.

**Review by psugne — APPROVED** [review]

**Comment by psugne** [comment]
No need for other reviewers.

**Pull request merged**